Add instructions on how to clean a specific room for Roborock (#27675)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

- Status information such as errors, clean time, consumables, etc.
- Viewing the camera
- Viewing the map

### How can I clean a specific room?

> We plan to make the process simpler in the future, but for now it is a multi-step process.

Reviewer: ```suggestion
We plan to make the process simpler in the future, but for now, it is a multi-step process.
```

1) Enable debug logging for this integration and reload it.
2) Search your logs for 'Got home data' and then find the attribute rooms.

> 3) Write the rooms down, they have a name and 6 digit id

Reviewer: ```suggestion
3) Write the rooms down; they have a name and 6 digit ID.
```
I think ID is typically [capitalized](https://ahdictionary.com/word/search.html?q=ID)

> 4) Go to Developer Tools -> Services -> Vacuum: Send Command and select your vacuum as the entity and 'get_room_mapping' as the command

Reviewer: ```suggestion
4) Go to **Developer Tools** > **Services** > **Vacuum: Send Command**. Select your vacuum as the entity and 'get_room_mapping' as the command.
```

> 5) Go back to your logs and look at the response to get_room_mapping, this is a list of the 6 digit ids you saw earlier to 2 digit ids. In your original list of room names and 6 digit ids, replace the 6 digit id with its pairing 2 digit id.

Reviewer: ```suggestion
5) Go back to your logs and look at the response to `get_room_mapping`. This is a list of the 6-digit IDs you saw earlier to 2-digit IDs. In your original list of room names and 6-digit IDs, replace the 6-digit ID with its pairing 2-digit ID.
```
*This is a list of the 6-digit IDs you saw earlier to 2-digit IDs.*: I don't understand. Is the list mapping the 6-digit IDs to the 2-digit ID?

> 6) Now you have the 2 digit id that your vacuum uses to describe a room.

Reviewer: ```suggestion
6) Now, you have the 2-digit ID that your vacuum uses to describe a room.
```

> 7) Go back to Developer Tools -> Services -> Vacuum: Send Command then type in app_segment_clean as your command and 'segments' with a list of the two digit ids you want to clean and then 'repeats' with a number 1->3 to determine how many times you want to clean these areas.

Reviewer: ```suggestion
7) Go back to **Developer Tools** > **Services** > **Vacuum: Send Command** then type `app_segment_clean` as your command and 'segments' with a list of the 2-digit IDs you want to clean. Then, add  `repeats` with a number (ranging from 1 to 3) to determine how many times you want to clean these areas.
```
